libexec/cli/pull.exec

```diff
@@ -86,7 +86,7 @@ case "$IMPORT_URI" in
     ;;
     shub://*)
         IMAGE_SHUB_ID=`echo "$IMPORT_URI" | sed -e 's@^shub://@@'`
-        exec $SINGULARITY_libexecdir/singularity/python/cli.py --shub $IMAGE_SHUB_ID
+        exec $SINGULARITY_libexecdir/singularity/python/cli.py --shub $IMAGE_SHUB_ID --pull-folder $PWD
```

Comment:
Ahh, looking at the API I think answered it for me. Where the image will land before it is run?